File: app/src/main/java/fr/frogdevelopment/nihongo/dico/data/rest/Entry.java
package fr.frogdevelopment.nihongo.dico.data.rest;

import android.text.SpannableString;

import com.fasterxml.jackson.annotation.JsonIgnore;

import java.io.Serializable;

public class Entry implements Serializable {

    private static final long serialVersionUID = 1L;

    public String senseSeq;
    public String kanji;
    public String kana;
    public String vocabulary;

    @JsonIgnore
    public SpannableString kanjiSpannable;
    @JsonIgnore
    public SpannableString kanaSpannable;
    @JsonIgnore
    public SpannableString vocabularySpannable;
}

File: app/src/main/java/fr/frogdevelopment/nihongo/dico/data/rest/Entry.kt
package fr.frogdevelopment.nihongo.dico.data.rest

import android.text.SpannableString
import com.fasterxml.jackson.annotation.JsonIgnore
import java.io.Serializable

class Entry(
        val senseSeq: String,
        val kanji: String?,
        val kana: String,
        val vocabulary: String
) : Serializable {

    @JsonIgnore
    var kanjiSpannable: SpannableString? = null
    @JsonIgnore
    var kanaSpannable: SpannableString? = null
    @JsonIgnore
    var vocabularySpannable: SpannableString? = null
}